nifty-script

```diff
@@ -423,69 +263,28 @@ verify_coverage_improvement() {
 # at least the same or better, a new "travis-tested" tag will be
 # pushed out to the app's repository.
 #####
-nifty_test_app() {
-
-    make reqs
-
-    make travis-tests
-
-    verify_coverage_improvement
-
-    # Do not tag unless we are the master branch and this is not a PR
-    if [[ "$TRAVIS_BRANCH" != "master" ||
-          "$TRAVIS_PULL_REQUEST" = "true" ]]; then
-        return
-    fi
-
-    # Also, do not tag if this is not a private repository. We detect
-    # private repositories by looking for a "source_key" to be defined
-    # in the .travis.yml file
-    if ! grep -q ^source_key .travis.yml; then
-        return
-    fi
-
-    __configure_git_authorship
-
-    # Tag the current commit
-    git tag -fm "Travis tested master" travis-tested
-    git push origin +travis-tested
-}
-
-# Main function
 
 # Ensure all expected aspects of the nifty environment are in place,
 # (git-based cache for travis test results, latest version of pip,
 # Xvfb server running and correcti DISPLAY environment variable set,
 # etc.)
 __init_nifty_environment
 
-# Test that we are invoked with an argument, (rather than the old
-# approach we were using of sourcing nifty-script). Give a useful
-# error message to help the developer massage their .travis.yaml
-if [ "$#" -lt 1 ]; then
-    echo "nifty-script should now be called with an argument giving the"
-    echo "test mode (rather than being sourced). Fix up .travis.yaml"
-    echo "to look like one of the following as appropriate:"
```

Comment:
Hmm.... do we still need code like this?

That is, perhaps we still have some apps that have a .travis.yml that first source nifty-script and then attempt to call the nifty_test_app function?

Instead of just wondering, I can check for them:

```
$ grep 'source.*nifty' */.travis.yml | grep -v ^site | sed -e 's,/\.travis.*,,'
cmsplugin-bootstrap-carousel
cmsplugin-collapse
cmsplugin-forms-builder
cmsplugin-tabs
django-shop-richcatalog
django-shop-richproduct
user-profile
```

OK, so there are only a handful left, and perhaps some of those we are ready to drop anyway?